examples/naming-conventions.md

```diff
@@ -0,0 +1,25 @@
+# Naming Conventions
+
+In this example, we would like to share some simple rules to apply some naming conventions in your project.
+
+Naming conventions can help the team to create and rename new files with a common shared standard.   
+
+In the following rules, written into `phparkitect.php` file into the root of a project, we said that:
```

Comment:
That would be the case for every example, should we repeat it every time? 🤔

Reply:
ok, do you think to remove all the seven line? 
Do you have some suggestions @micheleorselli ?

Reply:
What about:
```
In this example, the desired rules are:
# ...
Which can be translated in the configuration like this:
```

Reply:
Sounds good to me! 👍 
@micheleorselli ?

Reply:
yup, good to me!